docs/asset_fields_overview.md

```diff
@@ -0,0 +1,44 @@
+## Asset fields overview
+
+Asset Manager stores a a number of fields whose values work together to enforce how the asset is served.
+These fields are closely related to various document states from the publishing applications.
+Whitehall uses most of the fields, since it implements a full "editioned" flow. The following examples are thus based on Whitehall's behaviour.
```

Comment:
I'd probably drop this line about Whitehall - we should have some docs in Whitehall about how it interacts with Asset Manager (I think Korny wrote some).